l5kit/l5kit/tests/rasterization/render_context_test.py

```diff
@@ -14,11 +16,14 @@ def test_transform_points_to_raster() -> None:
         raster_size_px=image_shape_px,
         pixel_size_m=pixel_size_m,
         center_in_raster_ratio=center_in_raster_ratio,
-        set_origin_to_bottom=False,
+        set_origin_to_bottom=set_origin_to_bottom,
     )
 
     input_points = np.array([[0, 0], [10, 10], [-10, -10]])
-    expected_output_points = np.array([[100, 102], [110, 112], [90, 92]])
+    if not set_origin_to_bottom:
```

Comment:
Minor
```suggestion
    if set_origin_to_bottom:
```
And swap the Line 24 & 26